_benchmark/user-guide/concepts.md

```diff
@@ -20,95 +24,57 @@ Before using OpenSearch Benchmark, familiarize yourself with the following conce
+### Processing time
 
-#### Example A
+**Processing time** accounts for any extra overhead tasks that OpenSearch Benchmark performs during the life cycle of a request, such as setting up a request context manager and calling a method to pass the request off to the OpenSearch client. This is in contrast to **service time**, which only accounts for the different from when a request was sent and when the response is received by the OpenSearch client.
 
-The following diagrams illustrate how latency is calculated with an expected request response time of 200ms and the following settings: 
+### Took time
 
-- `search-clients` is set to `1`. 
-- `target-throughput` is set to `1` operation per second.
+**Took time** measures the amount of time the cluster spends processing a request on the server-side. The does not include the time it took for the request to reach the cluster from the client, or the time it took for the response to get back from the client to the cluster.
 
-<img src="{{site.url}}{{site.baseurl}}/images/benchmark/latency-explanation-1.png" alt="">
-
-When a request takes longer than 200ms, such as when a request takes 1110ms instead of 400ms, OpenSearch Benchmark sends the next request that was supposed to occur at 4.00s based on the `target-throughput` at 4.10s. All subsequent requests after the 4.10s request attempt to resynchronize with the `target-throughput` setting.
-
-<img src="{{site.url}}{{site.baseurl}}/images/benchmark/latency-explanation-2.png" alt="">
-
-When measuring the overall latency, OpenSearch Benchmark includes all performed requests. All requests have a latency of 200ms, except for the following two requests:
-
-- The request that lasted 1100ms. 
-- The subsquent request that was supposed to start at 4:00s. This request was delayed by 100ms, denoted by the orange area in the following diagram, and had a response time of 200ms. When calculating the latency for this request, OpenSearch Benchmark will account for the delayed start time and combine it with the response time. Thus, the latency for this request is **300ms**.
-
-<img src="{{site.url}}{{site.baseurl}}/images/benchmark/latency-explanation-3.png" alt="">
-
-#### Example B
-
-In this example, OpenSearch Benchmark assumes a latency of 200ms and uses the following latency settings:
-
-- `search_clients` is set to `1`.
-- `target-throughput` is set to `10` operations per second.
-
-The following diagram shows the schedule built by OpenSearch Benchmark with the expected response times.
+### Service time
 
-<img src="{{site.url}}{{site.baseurl}}/images/benchmark/b-latency-explanation-1.png" alt="">
+OpenSearch Benchmark does not have insight into how long OpenSearch takes to process a request, apart from extracting the [took time](#took-time) for the request. In OpenSearch, **service time** tracks the amount of time between when OpenSearch issues a request and receives a response.
```

Comment:
```suggestion
OpenSearch Benchmark does not have insight into how long OpenSearch takes to process a request, apart from extracting the [took time](#took-time) for the request. **Service time** measures the time from when OpenSearch issues a request until it receives a response.
```

Reply:
Although "measures" implies a length or amount of time, I think removing "amount of time" from the sentence makes it more confusing, since time can mean multiple types of measurements, for example, the clock time a request was issued, versus the length of time between two requests.